### DISCUSSION TITLE
Any examples integrating jupyter-book and sphinx gallery?

### DISCUSSION THREAD

**Discussion by jni** [executablebooks/meta#434] [Q&A]
Hello! Has anyone built a site with JupyterBook that includes a sphinx-gallery as a section? We loved building our site with JupyterBook for [napari](https://napari.org/index.html) but now we'd like to also add a gallery of all our [code examples](https://github.com/napari/napari/tree/529181d20efdcf300233dd89722c99f37dbe5ffd/examples), and I'm not sure where to start...

**Comment by choldgraf** [comment]
Could you give it a shot by manually activating sphinx_gallery via custom Sphinx config? here are some docs: https://jupyterbook.org/advanced/sphinx.html
